src/components/RadioButtonFieldGroup.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from "react";
+import {
+  View,
+  StyleProp,
+  ViewStyle,
+  TextStyle,
+  StyleSheet,
+} from "react-native";
+import Text from "./Text";
+import RadioButtonFieldRow from "./RadioButtonFieldRow";
+import Divider from "./Divider";
+
+interface RadioButtonOption {
+  key: string;
+  value: string;
+}
+
+interface Props {
+  options: RadioButtonOption[];
+  label?: string;
+  onSelect?: (value: string) => void;
+  value?: string;
+  radioButtonColor?: string;
```

Comment:
the colors should all be using the theme values, ie: "primary", "secondary", etc.